lib/dependencies/ImportParserPlugin.js

```diff
@@ -205,6 +207,20 @@ class ImportParserPlugin {
 				mode = "lazy";
 			}
 
+			const referencedPropertiesInDestructuring =
+				parser.destructuringAssignmentPropertiesFor(expr);
+			if (referencedPropertiesInDestructuring) {
+				if (exports) {
+					parser.state.module.addWarning(
+						new UnsupportedFeatureWarning(
```

Comment:
@chenxsan I'm pinging you here for this one this will likely be a documentation note for this feature. I'll try to remember to reference the generated issue.

Reply:
This should likely be added to the `Module Methods` or `magic comments` section where `/* webpackExports */` is used.